README.rst

```diff
@@ -240,6 +240,59 @@ inside the current folder. If you omit the final ``tmp/`` you'll leave the
 empty ``tmp`` directory in place. (**WARNING!** Don't put the ``.`` *after*
 the ``--erase`` option! Obviously, your project files will all be deleted.)
 
+🏷️ Git-clean integration
```

Comment:
Add the emoji on the right end of the section title, just like the other sections have it.